### DISCUSSION TITLE
Rule "BR-BT-00262-0211" in SDK 1.9 does not implement article 13 of 2014/24/EU

### DISCUSSION THREAD

**Discussion by RolfKewitz** [OP-TED/eForms-SDK#722] [Q&A] [answered]
If rule „BR-BT-00262-0211“ shall implement article 13 of 2014/24/EU it has to be changed.

**In current version of the rule ="BR-BT-00262-0211" in SDK 1.9 
a 'buyer-legal-type of  ('org-sub','org-sub-cga','org-sub-ra','org-sub-la') may not publish a notice by 
using a NoticeSubType of ('1','4','7','8','9','10','12','16','17','18','19','20','23','25','29','33','36','38')
for CPV-Codes 0-44 or 48 although this would be in accordance with the law!**

Correct version of the rule should be:

> \<assert id="BR-BT-00262-0211" role="ERROR" diagnostics="ND-ProcedureProcurementScope_BT-262-Procedure" 
test="(
not(
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '45') or 
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '49') or
 starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '5') or 
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '6') or 
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '7') or 
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '8') or 
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '9')) 
or
(../ext:UBLExtensions/ext:UBLExtension/ext:ExtensionContent/efext:EformsExtension/efac:NoticeSubType/cbc:SubTypeCode/normalize-space(text()) = ('1','4','7','8','9','10','12','16','17','18','19','20','23','25','29','33','36','38')
)
or
not(
 (../cac:ContractingParty/cac:ContractingPartyType/cbc:PartyTypeCode[@listName='buyer-legal-type']/normalize-space(text()) = ('org-sub','org-sub-cga','org-sub-ra','org-sub-la')) 
)
and
 (cac:MainCommodityClassification/cbc:ItemClassificationCode)
)">rule|text|BR-BT-00262-0211</assert>
> 

instead of current version in SDK 1.9 which seems to be wrong:

> \<assert id="BR-BT-00262-0211" role="ERROR" diagnostics="ND-ProcedureProcurementScope_BT-262-Procedure" 
test="(
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '45') or 
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '49') or
 starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '5') or 
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '6') or 
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '7') or 
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '8') or 
starts-with(cac:MainCommodityClassification/cbc:ItemClassificationCode/normalize-space(text()), '9')) or
not( (../ext:UBLExtensions/ext:UBLExtension/ext:ExtensionContent/efext:EformsExtension/efac:NoticeSubType/cbc:SubTypeCode/normalize-space(text()) = ('1','4','7','8','9','10','12','16','17','18','19','20','23','25','29','33','36','38')) 
and
 (../cac:ContractingParty/cac:ContractingPartyType/cbc:PartyTypeCode[@listName='buyer-legal-type']/normalize-space(text()) = ('org-sub','org-sub-cga','org-sub-ra','org-sub-la')) 
and
 (cac:MainCommodityClassification/cbc:ItemClassificationCode)
)">rule|text|BR-BT-00262-0211</assert>

**Comment by YvesJo** [comment, marked as answer]
Your raised concern and understanding of how subsidized organizations should be interpreted have reached the business people and are currently under evaluation. The issue is first of all Business Related and not directly SDK related.
